colabs/huggingface/LLM_Finetuning_Notebook.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    entity = "wandb" #@param
</pre><p>I'd write a note to explicitly ask users to change the entity to their username to prevent confusion if they just press run all</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>